pkg/routeagent_driver/handlers/mtu/mtuhandler.go

```diff
@@ -274,29 +276,52 @@ func (h *mtuHandler) Stop(uninstall bool) error {
 	return nil
 }
 
-func (h *mtuHandler) forceMssClamping(endpoint *submV1.Endpoint) error {
-	defaultHostIface, err := netlinkAPI.GetDefaultGatewayInterface()
-	if err != nil {
-		return errors.Wrapf(err, "Unable to find the default interface on host")
-	}
+func (h *mtuHandler) forceMssClamping(endpoint *submV1.Endpoint, usrForceMssVal int) error {
+	if usrForceMssVal == 0 {
+		defaultHostIface, err := netlinkAPI.GetDefaultGatewayInterface()
+		if err != nil {
+			return errors.Wrapf(err, "Unable to find the default interface on host")
+		}
 
-	overHeadSize := maxIpsecOverhead
-	if endpoint.Spec.Backend == vxlan.CableDriverName {
-		overHeadSize = vxlan.VxlanOverhead
-	}
+		overHeadSize := maxIpsecOverhead
+		if endpoint.Spec.Backend == vxlan.CableDriverName {
+			overHeadSize = vxlan.VxlanOverhead
+		}
 
-	klog.Infof("forceMssClamping to: IF_MTU(%d)-Overhead(%d)=%d",
-		defaultHostIface.MTU, overHeadSize, (defaultHostIface.MTU - overHeadSize))
+		usrForceMssVal = defaultHostIface.MTU - overHeadSize
+	}
 
+	klog.Infof("forceMssClamping to: %d", usrForceMssVal)
```

Comment:
Please add a log for us to know if the MSSClamp value is the default one or something that was forced by user. As we will mostly be relying on the logs while debugging any issue.

Reply:
Yeah, that's a good idea.
Done